### PR TITLE
Smashing Magazine Heatmap

### PR DESCRIPTION
**Pull request by thneves**
The goal of this project is to create an hierarchical heat map color of the Smashing Magazine webpage.
We used 'divs' represent each element with its own color of importance.
There are 3 types of color, darker, medium and light representing the importance of the elements.
We followed the original template on the width of 1440px.
There is some minor hover effects to rotate some elements like in the original page.

### CODE REVIEW

**Comment by thneves** [comment]
we are facing a problem with the Lighthouse, but we couldn't fix it.

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required

### Hello team,

**Good job on your project.**

**Please see the required changes below:**

## Mandatory

1). **All the elements and element's backgrounds and font colors should be grayscale color (black, white or any shade of grey). You used shades of blue and magenta. For shades of grey, you can reference the table below:**
shades

2). **Add text on your webpage to help with the visual weight and feel of the page. The screenshot below is from odin.**

odin

3). **There is a requirement to have an article tag which takes full width. (Hint: you can change a section tag/ div tag, which takes full width, into an article tag).**

4). **Your linter files are not set up properly. That is why you are having check failures. First,
run `npm init`, to set up a package.json file, then follow these microverse [guideline](https://github.com/microverseinc/linters-config/tree/master/html-css)s on how to set up a linter for HTML & CSS.**

**Happy coding 😃.**

**PS: You can reach me on slack(Rose Wanjohi).**

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__I can see that great effort has been put in place to make changes required by the previous reviewer. However, we still have one more changes to make before this project can be approved. Please check my points below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] As mentioned by the previous reviewer, you need to add a `1 article row` section which will contain one article and will take the full width of the page. You already have this 1 article container, but you need to use the `<article>` tag instead of either `<div>` or `<section>` tag.





- [x] Although the content is not important, your footer section items should be linked.



As a result of the above requirements, the items selected in the image below should be linked.



**Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: Hello Team!! Great job working on your project I see the effort you have put into it! :clap: :clap:

## Required Changes
### [Mandatory] :red_circle: 

- [ ] Use semantic HTML for the navbar and build it with an unordered list instead of the ```<div>``` containers. See inline comments

- [ ] Most of the website elements are not visually delimited and heat-maped. This makes it hard to identify where each one of the elements/sections starts and ends. We should improve this.
image
image
image

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

File: index.html
Comment (jubaan) [inline index.html]: - [ ] This should use ```<nav>``` semantic HTML. Create it using ```<ul>```, be aware of the elements that need to are links and use the link HTML tag for them.

**Review by jubaan — CHANGES_REQUESTED** [review]

File: index.html
Comment (acushlakoncept) [inline index.html]: Semantically, the content of `ul` elements should have `li` elements: Your code should look like this:
```
<ul class="header-list d-flex">
        <li><a class="list-item bg-medium" href="#"></a></li>
       .....
</ul>
```

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

#### Hello, @julian3493 and @thneves,

Nice job so far, let's address the changes below and request for another code review request.

- [ ] Fix the HTML validation warnings/errors, use the [w3 Validator](https://validator.w3.org/)

- [ ] Make sure you fix all lint errors before submitting another code review request
Screen Shot 2020-09-29 at 9 35 13 PM

I have made a comment on your code below, do check it as well.

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: Hello!! Great job working on the previously requested changes.

### [Mandatory] :red_circle: 

- [ ] Footer menu should be an ```<ul>``` with ```<li>``` elements that wrap ```<a>``` elements in the same way as the navbar menu elements. See inline comments

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Comment by thneves** [comment]
> # Status: Changes Required recycle
> ## wave wave Hello!! Great job working on the previously requested changes.
> ### [Mandatory] red_circle
> 
>     * [ ]  Footer menu should be an `<ul>` with `<li>` elements that wrap `<a>` elements in the same way as the navbar menu elements. See inline comments
> 
> 
> ## Keep up the great work muscle and Happy Coding smiley computer!!

So why you didn't say it in the previous review ?

File: index.html
Comment (jubaan) [inline index.html]: Example:
```
<ul class="...some clases..">
  <li class="...some classes..."><a href="#">some link</a></li>
  <li class="...some classes..."><a href="#">some link</a></li>
  <li class="...some classes..."><a href="#">some link</a></li>
  <li class="...some classes..."><a href="#">some link</a></li>
</ul>
```

**Review by jubaan — CHANGES_REQUESTED** [review]

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

## Congratulations team :tada: :tada: 

## This project has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:

You can now merge this branch to master.
Good luck with your next project.

**I'm @taofeekolalere on slack.
Keep up the good work :man_technologist: :muscle:**

**Pull request merged**
